.github/workflows/app-build.yml

```diff
@@ -13,43 +13,17 @@ jobs:
   app:
     name: Build audit and test
     runs-on: ubuntu-latest
-    defaults:
-      run:
-        working-directory: ./app
     env:
       PUBLIC_SPOTIFY_CLIENT_ID: ${{ secrets.SPOTIFY_CLIENT_ID }}
       PUBLIC_DEMO_JSON_URL: ${{ vars.PUBLIC_DEMO_JSON_URL }}
-
     steps:
       - name: Checkout code
         uses: actions/checkout@v6
-
-      - name: Install pnpm
-        uses: pnpm/action-setup@v4
-        with:
-          run_install: false
-
-      - name: Use Node 24
-        uses: actions/setup-node@v6
         with:
-          node-version: "24"
-          cache: "pnpm"
-
-      - name: Install dependencies
-        run: pnpm install
-
-      - name: Audit dependencies
-        continue-on-error: true
-        run: pnpm audit --prod
-
-      - name: Check format
-        run: pnpm run format:check
-
-      - name: Check Lint
-        run: pnpm run lint
+          fetch-depth: 0
 
-      - name: Check test
-        run: pnpm run test:coverage
+      - name: Install moon
+        uses: moonrepo/setup-moon-action@v1
```

Comment:
This GH Actions seems to be deprecated: https://github.com/moonrepo/setup-moon-action?tab=readme-ov-file